pkg/volume/csi/expander.go

```diff
@@ -94,12 +94,30 @@ func (c *csiPlugin) nodeExpandWithClient(
 		return false, nil
 	}
 
-	volumeTargetPath := resizeOptions.DeviceMountPath
+	pv := resizeOptions.VolumeSpec.PersistentVolume
+	if pv == nil {
+		return false, fmt.Errorf("Expander.NodeExpand failed to find associated PersistentVolume for plugin %s", c.GetPluginName())
+	}
+
+	opts := csiResizeOptions{
+		volumePath:        resizeOptions.DeviceMountPath,
+		stagingTargetPath: resizeOptions.DeviceStagePath,
+		volumeid:          csiSource.VolumeHandle,
+		newSize:           resizeOptions.NewSize,
+		fsType:            csiSource.FSType,
+		accessMode:        api.ReadWriteOnce,
+		mountOptions:      pv.Spec.MountOptions,
+	}
 	if !fsVolume {
-		volumeTargetPath = resizeOptions.DevicePath
+		opts.volumePath = resizeOptions.DevicePath
```

Comment:
why rename `DevicePath` to `VolumePath` here, if they are referring to the same thing should they have the same name?

Reply:
This is because this field is called `volume_path` in CSI but internally in k8s we call it `DevicePath`